### PR TITLE
8169098: [TestBug] Manual test case for JDK-8089915

### PR DESCRIPTION
**Pull request by rahulk178**
Adding a new manual test case for accept attribute of input tag.
This test is to verify the fix for [JDK-8089915](https://bugs.openjdk.java.net/browse/JDK-8089915) .
We have tested the test cases on all three platforms and its working as expected.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8169098](https://bugs.openjdk.java.net/browse/JDK-8169098): [TestBug] Manual test case for JDK-8089915

### Reviewers
 * [Kevin Rushforth](https://openjdk.java.net/census#kcr) (@kevinrushforth - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jfx pull/607/head:pull/607` \
`$ git checkout pull/607`

Update a local copy of the PR: \
`$ git checkout pull/607` \
`$ git pull https://git.openjdk.java.net/jfx pull/607/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 607`

View PR using the GUI difftool: \
`$ git pr show -t 607`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jfx/pull/607.diff">https://git.openjdk.java.net/jfx/pull/607.diff</a>

</details>

### CODE REVIEW

File: tests/manual/web/InputTypeAcceptAttributeTest.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+
```
Comment (kevinrushforth) [inline tests/manual/web/InputTypeAcceptAttributeTest.java:26]: Minor: can you remove the extra blank line?

File: tests/manual/web/InputTypeAcceptAttributeTest.java [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+import javafx.application.Platform;
+import javafx.geometry.Insets;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.web.WebEngine;
+import javafx.scene.web.WebView;
+import javafx.stage.Stage;
+import javafx.scene.layout.VBox;
+import javafx.scene.layout.HBox;
```
Comment (kevinrushforth) [inline tests/manual/web/InputTypeAcceptAttributeTest.java:37]: Suggestion: maybe sort the list of imports?

**Review by kevinrushforth — COMMENTED** [review]
The test looks good to me. I left two minor formatting comments.

File: tests/manual/web/InputTypeAcceptAttributeTest.java (hunk 2 above)
Comment (rahulk178) [reply tests/manual/web/InputTypeAcceptAttributeTest.java]: I have updated the review comments.

File: tests/manual/web/InputTypeAcceptAttributeTest.java (hunk 1 above)
Comment (rahulk178) [reply tests/manual/web/InputTypeAcceptAttributeTest.java]: I have updated the review comments.

**Review by kevinrushforth — APPROVED** [review]
Looks good.

**Pull request closed without merging**